pylabrobot/resources/eppendorf/plates.py

```diff
@@ -0,0 +1,108 @@
+""" Thermo Fisher & Thermo Fisher Scientific plates """
+
+# pylint: disable=invalid-name
+
+from pylabrobot.resources.well import Well, WellBottomType, CrossSectionType
+from pylabrobot.resources.utils import create_ordered_items_2d
+from pylabrobot.resources.plate import Lid, Plate
+
+from pylabrobot.resources.height_volume_functions import (
+  calculate_liquid_height_in_container_2segments_round_vbottom,
+  calculate_liquid_volume_container_2segments_round_vbottom
+  )
+
+
+# # # # # # # # # # Thermo_AB_96_wellplate_300ul_Vb_EnduraPlate # # # # # # # # # #
+
+def _compute_volume_from_height_Eppendorf_96_wellplate_250ul_Vb(h: float):
+  if h > 20.3:
+    raise ValueError(f"Height {h} is too large for" + \
+                     "Eppendorf_96_wellplate_250ul_Vb")
+  return calculate_liquid_volume_container_2segments_round_vbottom(
+    d=6.74,
+    h_cone=8.54,
+    h_cylinder=10.96,
+    liquid_height=h)
+
+
+def _compute_height_from_volume_Eppendorf_96_wellplate_250ul_Vb(liquid_volume: float):
+  if liquid_volume > 262.5: # 5% tolerance
+    raise ValueError(f"Volume {liquid_volume} is too large for" + \
+                     "Eppendorf_96_wellplate_250ul_Vb")
+  return round(calculate_liquid_height_in_container_2segments_round_vbottom(
+    d=6.74,
+    h_cone=8.54,
+    h_cylinder=10.96,
+    liquid_volume=liquid_volume),3)
+
+# results_testing_optimal_arguments_dict = {
+#     "Volume (ul)": [0, 4, 8, 20, 70, 120, 170, 220, 260],
+#     "Observed Height (mm)": [0, 0.45, 1.45, 2.55, 6.45, 9.05, 11.6, 13.15, 14.349],
+#     "Predicted Height (mm)": [0.0, 2.91, 3.66, 4.97, 7.54, 9.06, 10.46, 11.86, 12.98],
```

Comment:
there is quite a big difference between predicted and observed height. do you think we should use your polynomial fit again?

Reply:
Done